webpack.renderer.config.js

```diff
@@ -7,7 +7,9 @@ const webpack = require(`webpack`)
 const fs = require(`fs`)
 
 const HtmlWebpackPlugin = require(`html-webpack-plugin`)
+const CSPWebpackPlugin = require(`csp-webpack-plugin`)
 const VueLoaderPlugin = require(`vue-loader/lib/plugin`)
+const MiniCssExtractPlugin = require(`mini-css-extract-plugin`)
 // const BundleAnalyzerPlugin = require(`webpack-bundle-analyzer`)
```

Comment:
delete comments?

Reply:
can be uncommented to actually analyze the bundle size, which we probably want to do from time to time